### ISSUE TITLE
Missing role="img" in Toolbar Images Affecting Accessibility

### ISSUE THREAD

**Issue by diogogasparr** [stephanrauh/ngx-extended-pdf-viewer#1838]
**Version**: 18.0.0-beta.1

**Description**:
Several toolbar images do not have the role="img" attribute, which is crucial for compatibility with accessibility applications and tools.

**Example**:
`<svg style="width: 24px; height: 24px" viewBox="0 0 24 24"></svg>`

**Expected**:
All SVG images used in toolbars should include the role="img" attribute to ensure that accessibility tools can interpret them correctly.

**Comment by diogogasparr** [comment]
Also, adding to this - it's also missing a text alternative for each toolbar svg

**Comment by stephanrauh** [comment]
Good point. Now I'm happy I delayed publishing the final version! Accessibility is important to me.

**Comment by stephanrauh** [comment]
@stephanrauh I'm no expert an the topic, so I need your help. Can you check [my commit](https://github.com/stephanrauh/ngx-extended-pdf-viewer/commit/404bcc8f359873713d501e6ed643d2c929cad41d) and tell me if I got it right, please?

**Comment by stephanrauh** [comment]
I've published the improvement with version 18.0.0-beta.4, so you can test your requirements with real code now.

**Comment by diogogasparr** [comment]
Still missing in these:
`<button type="button" class="toolbarButton hiddenMediumView" id="openFile" title="Open File" aria-label="Open File">
    <svg style="width: 24px; height: 20px" viewBox="0 0 24 24">
    </svg>
</button>`
(and others in tab)

and in viewerContainer:
`<div id="viewerContainer" [style.top]="viewerPositionTop" [style.backgroundColor]="backgroundColor"
              tabindex="0">`
missing role="document"

**Comment by stephanrauh** [comment]
OK, I understand something's missing. TBH, I already suspected so. But I didn't manage to wrap around my head around the accessability documentation. Lots and lots of text, so I felt a bit lost. What's the correct role for a button? ChatGPT tells me I don't have to add `role="button"` because I'm already using the semantic `<button>` tag.

As for `role="document"`: consider it done!

**Comment by stephanrauh** [comment]
I've added `role="button"` and `role="document"` in version 18.0.0-beta.5. Hope that help. As I've mentioned, I'm no expert, and I can't wrap my head around the accessibility documentation. That's annoying, because it's an important topic, and I've tried several times over the last decade.
